detox/src/devices/Device.js

```diff
@@ -17,7 +17,7 @@ class Device {
     sessionConfig,
     runtimeErrorComposer,
   }) {
-    cutStackTraces(this, [
+    wrapWithStackTraceCutter(this, [
```

Comment:
This is great ❤️
Could we get rid of the methods list, too? :prettyplease:

Reply:
No TypeScript, no candies, sorry.

Reply:
Lol...
A simple `filter()` lambda with a distinction between "public" and "private" (`startsWith('_')`) predicate?!